src/TestExplorer/TestRunner.ts

```diff
@@ -208,6 +157,8 @@ export class TestRunner {
             // will rebuild everything again
             if (testKind !== TestKind.coverage) {
                 const task = await getBuildAllTask(this.folderContext);
+                task.definition.dontTriggerTestDiscovery = true;
```

Comment:
Is this still needed in pre 6.0 where we are getting the test list from `swift test list`

Reply:
I actually think this can be removed